frb_example/dart_minimal/rust/Cargo.toml

```diff
@@ -7,7 +7,21 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-flutter_rust_bridge = { path = "../../../frb_rust" }
+flutter_rust_bridge = { path = "../../../frb_rust", default-features = false, features = [
+    "anyhow",
+    # Deliberately remove this feature to ensure frb can work without it
+    # "dart-opaque",
+    "log",
+    "portable-atomic",
+    "rust-async",
+    "thread-pool",
+    "user-utils",
+    "wasm-start",
+] }
 
 [lints.rust]
 unexpected_cfgs = { level = "warn", check-cfg = ['cfg(frb_expand)'] }
+
+[features]
+default = ["dart-opaque"]
+dart-opaque = ["flutter_rust_bridge/dart-opaque"]
```

Comment:
nit: what about naming it `flutter-rust-bridge--dart-opaque` or something like that, so that it is more clear we are forwarding to there when we read our test infra code.

Reply:
Oh yes, the "EDIT" above is the last one, so we may use that approach etc.